src/commands/create.js

```diff
@@ -112,34 +117,36 @@ export default asyncCommand({
 
 		spinner.text = 'Installing dev dependencies';
```

Comment:
super nitpick (feel free to ignore) - seems like the dev dep message and "done!" message should be inside the if here.